docs/tutorials/index.md

```diff
@@ -22,34 +22,33 @@ title: "Quickstart (local)"
   ~ under the License.
   -->
 
-This quickstart gets you started with Apache Druid and introduces you to Druid ingestion and query features. For this tutorial, we recommend a machine with at least 6 GB of RAM.
+This quickstart helps you install Apache Druid and introduces you to Druid ingestion and query features. For this tutorial, you need a machine with at least 16 GiB of RAM.
 
-In this quickstart, you'll do the following:
+In this quickstart, you'll:
 - install Druid
 - start up Druid services
 - use SQL to ingest and query data
 
 Druid supports a variety of ingestion options. Once you're done with this tutorial, refer to the
 [Ingestion](../ingestion/index.md) page to determine which ingestion method is right for you.
 
-## Requirements
+## Prerequisites
 
 You can follow these steps on a relatively modest machine, such as a workstation or virtual server with 16 GiB of RAM.
 
 Druid comes equipped with launch scripts that can be used to start all processes on a single server. Here, we will use [`auto`](../operations/single-server.md#druid-auto-start), which automatically sets various runtime properties based on available processors and memory.
 
-In addition, Druid includes several [bundled non-automatic profiles](../operations/single-server.md) for a range of machine sizes. These range from nano (1 CPU, 4GiB RAM) to x-large (64 CPU, 512GiB RAM). 
-We won't use those here, but for more information, see [Single server deployment](../operations/single-server.md). For additional information on deploying Druid services across clustered machines, see [Clustered deployment](./cluster.md).
+In addition, Druid includes several [bundled static profiles](../operations/single-server.md) for a range of machine sizes. These range from nano (1 CPU, 4 GiB RAM) to x-large (64 CPU, 512 GiB RAM).
+See [Single server deployment](../operations/single-server.md) for more information. To learn how to deploy Druid services across clustered machines, see [Clustered deployment](./cluster.md).
 
 The software requirements for the installation machine are:
 
 * Linux, Mac OS X, or other Unix-like OS. (Windows is not supported)
 * [Java 8u92+ or Java 11](../operations/java.md)
-* [Python2 or Python3](../operations/python.md)
+* Python 2 or Python 3
 
-> Druid relies on the environment variables `JAVA_HOME` or `DRUID_JAVA_HOME` to find Java on the machine. You can set
-`DRUID_JAVA_HOME` if there is more than one instance of Java. To verify Java requirements for your environment, run the 
-`bin/verify-java` script.
+You must set either the `JAVA_HOME` or `DRUID_JAVA_HOME` environment variable for Druid to find Java on the machine. If there is more than one version of Java installed, set
+`DRUID_JAVA_HOME`. You can run `bin/verify-java` to verify Java requirements for your environment.
```

Comment:
```suggestion
`DRUID_JAVA_HOME`. You can run `apache-druid-{{DRUIDVERSION}}/bin/verify-java` to verify Java requirements for your environment.
```
@paul-rogers , @vtlim mentioned this, but when you unzip the distribution it goes into the folder named apache-druid-{{DRUIDVERSION}} . The docs build script replaces druid {{DRUIDVERSION}} with the value. For example: `apache-druid-25.0`